content/millennium/dstu2/individuals.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: Individuals
+title: Individuals | DSTU 2 API
 layout: api
 ---
 
-<%= render '/individuals.*' %>
```

Comment:
Same question

Reply:
as above, renamed existing html file to be dstu2 specific and added new one for r4